lib/checks/aria/aria-allowed-role.js

```diff
@@ -18,6 +19,10 @@ const unallowedRoles = axe.commons.aria.getElementUnallowedRoles(
 
 if (unallowedRoles.length) {
 	this.data(unallowedRoles);
+	if (!dom.isVisible(node, true)) {
+		// flag hidden elements for review
+		return null;
```

Comment:
This should return undefined. I know this also works, but we should do this consistently.